### DISCUSSION TITLE
TS Component example with Composition Api and Vuetify 3

### DISCUSSION THREAD

**Discussion by m4ss1m0g** [dm4t2/vue-currency-input#404] [Q&A] [answered]
Can you provide an example of a Component with Vue 3 and [Composition API](https://vuejs.org/guide/typescript/composition-api.html) with Typescript and Vuetify?

**Comment by dm4t2** [comment, marked as answer]
See #371. I will add this example to the [docs](https://dm4t2.github.io/vue-currency-input/examples.html).
